.devcontainer/README.md

```diff
@@ -12,6 +12,8 @@ CCCL uses [Development Containers](https://containers.dev/) to provide consisten
 ### Prerequisites
 - [Visual Studio Code](https://code.visualstudio.com/)
 - [Remote - Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+- [NVIDIA Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html)
```

Comment:
Hm, I'm realizing this list doesn't include `Docker`. I can't recall why I would have neglected to include that. Do you think we should include that for completeness too?

Reply:
Somehow I didn't notice that either 😅. I guess it's because I'm familiar with devcontainers and know that it's a requirement. For someone who's unfamiliar with docker and/or devcontainers, that won't be as obvious. I'll add it in!

Reply:
@jrhemstad The Remote - Containers extension (we should probably rename it to Dev Containers Extension btw) does install docker underneath. No need to add it explicitly.

Reply:
Oh, I didn't know it installs it for you. I can remove it